namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-community-accommodation-dev/resources/variables.tf

```diff
@@ -0,0 +1,48 @@
+
+variable "cluster_name" {
+}
+
+variable "vpc_name" {
+}
+
+variable "application" {
+  description = "Name of Application you are deploying"
+  default     = "Community Accommodation"
+}
+
+variable "namespace" {
+  default = "hmpps-community-accommodation-dev"
+}
+
+variable "business_unit" {
+  description = "Area of the MOJ responsible for the service."
+  default     = "HMPPS"
+}
+
+variable "team_name" {
+  description = "The name of your development team"
+  default     = "hmpps-community-accommodation"
+}
+
+variable "environment" {
+  description = "The type of environment you're deploying to."
+  default     = "development"
+}
+
+variable "infrastructure_support" {
+  description = "The team responsible for managing the infrastructure. Should be of the form team-email."
+  default     = "cas3@digital.justice.gov.uk"
+}
+
+variable "is_production" {
+  default = "false"
+}
+
+variable "slack_channel" {
+  description = "Team slack channel to use if we need to contact your team"
+  default     = "community-accommodation-service-tier-3-team"
```

Comment:
@pezholio If we can safely rename our dev Slack channel we can update this before it goes in? If we aren't confident should we rename the `approved-premises-team-dev` channel to `cas-dev` and then recreate `approved-premises-team-dev` just in case we get any messages send to it whilst we still have the old namespace in service?

Reply:
Yeah, I can do that now 👍